galleries/examples/images_contours_and_fields/contourf_demo.py

```diff
@@ -104,13 +96,36 @@
 fig, axs = plt.subplots(2, 2, layout="constrained")
 
 for ax, extend in zip(axs.flat, extends):
-    cs = ax.contourf(X, Y, Z, levels, cmap=cmap, extend=extend, origin=origin)
+    cs = ax.contourf(X, Y, Z, levels, cmap=cmap, extend=extend)
     fig.colorbar(cs, ax=ax, shrink=0.9)
     ax.set_title("extend = %s" % extend)
     ax.locator_params(nbins=4)
 
 plt.show()
 
+# %%
+# Customizing Contour Plots with the origin Keyword
+# -------------------------------------------------
+# This code showcases contour plot customization using the origin keyword,
+# demonstrating distinct upper and lower origin perspectives.
+
+x = np.arange(1, 10)
+y = x.reshape(-1, 1)
+h = x * y
+
+fig, (ax1, ax2) = plt.subplots(ncols=2)
+# Note: lower and upper are the values provided
+# to origin argument in contourf.
+for (ax, origin) in [(ax1, 'upper'), (ax2, 'lower')]:
+    ax.set_title(f"origin={origin}")
+    cs = ax.contourf(h, levels=[10, 30, 50], colors=[
+                     '#808080', '#A0A0A0', '#C0C0C0'], extend='both', origin=origin)
+    cs.cmap.set_over('red')
+    cs.cmap.set_under('blue')
+    cs.changed()
```

Comment:
You can simplify this to:

```suggestion
fig, (ax1, ax2) = plt.subplots(ncols=2)
ax1.set_title("origin='upper'")
ax2.set_title("origin='lower'")
cs = ax1.contourf(h, levels=np.arange(5, 70, 5), extend='both', origin="upper")
cs = ax2.contourf(h, levels=np.arange(5, 70, 5), extend='both', origin="lower")
```
![grafik](https://github.com/matplotlib/matplotlib/assets/2836374/9d7d9f2d-5f55-4d84-98c0-820641bdaa83)

Reply:
This look more optimized. Thanks, Let me add it.